api/models/graphite.go

```diff
@@ -0,0 +1,94 @@
+package models
+
+import (
+	"gopkg.in/raintank/schema.v1"
+	"sort"
+)
+
+type GraphiteRender struct {
```

Comment:
We could rename the package from "models" to "DTO" (data transfer objects), but i like models.

All of the types here are either input or output from the API.  THis is useful when integrating with the system as a client library can just import this package to be able to send correct payloads and decode the responses.   I dont think there is any need to split it up.

Req is only used internally now, but will be used externally when we add clustering.